.travis.yml

```diff
@@ -2,19 +2,22 @@ language: java
 dist: trusty
 sudo: true
 before_cache:
-  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+- rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
 cache:
   directories:
-    - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
-    - $HOME/.m2
+  - $HOME/.gradle/caches/
+  - $HOME/.gradle/wrapper/
+  - $HOME/.m2
 jdk:
-  - oraclejdk8
+- oraclejdk8
 script: ./gradlew jacocoTestReport ;
+env:
+- ARTIFACTORY_USERNAME=gatkci
+- secure: E0LWXgX3aWSE/DWHXXDx4vrAq4uX6vKg402wToaZ5otbHQ/UP0H7/FA5jomavAXoC46oMVHZcEltZ5OVhuJ0NW8yYxUCecJ1D/YvVQmnfFABcV/qLM+k4e2rYQOKVw/pejB2gG8XdTA+XE2WyTeENbmIkputS8f1ndKWCmZxuuk=
 after_success:
-  - echo "TRAVIS_BRANCH='$TRAVIS_BRANCH'";
-    echo "JAVA_HOME='$JAVA_HOME'";
-    ./gradlew coveralls;
-    if [ "$TRAVIS_BRANCH" == "master" ]; then
-       ./gradlew uploadArchives;
-    fi
+- echo "TRAVIS_BRANCH='$TRAVIS_BRANCH'"; echo "JAVA_HOME='$JAVA_HOME'";
+- bash <(curl -s https://codecov.io/bash); if [ "$TRAVIS_BRANCH" == "master" ]; then
+  ./gradlew uploadArchives; fi
+before_install:
+- openssl aes-256-cbc -K $encrypted_ba061aac1250_key -iv $encrypted_ba061aac1250_iv
```

Comment:
I just went back and tried it again and I think you're correct - I let travis do the update to the .yml file but I think I had added that line. So I think it can be removed, and I take back what I said about the keys - I think the encryption is specific to this repo. So anyway, yes, I think that line can be removed...